integration_tests/setup_test.go

```diff
@@ -461,11 +460,11 @@ func (s *IntegrationTestSuite) initGenesis() {
 	s.Require().NoError(err)
 	appGenState[gravitytypes.ModuleName] = bz
 
-	incentivesGenState := incentivestypes.DefaultGenesisState()
-	s.Require().NoError(cdc.UnmarshalJSON(appGenState[gravitytypes.ModuleName], &gravityGenState))
-	bz, err = cdc.MarshalJSON(&incentivesGenState)
-	s.Require().NoError(err)
-	appGenState[incentivestypes.ModuleName] = bz
+	// incentivesGenState := incentivestypes.DefaultGenesisState()
```

Comment:
If we don't need this code we should delete it. I also noticed on reading it again that it looks like the `NoError` check is using `gravityGenState` which would have been incorrect anyway.